.github/workflows/rule-validate.yml

```diff
@@ -92,6 +93,45 @@ jobs:
         id: get_head
         run: echo "##[set-output name=HEAD;]$(git rev-parse HEAD)"
 
+      - name: Get changed detection-rules
+        id: changed-files
+        uses: tj-actions/changed-files@v39
+        with:
+          files: "detection-rules/**"
+          recover_deleted_files: true
+
+      - name: "Find updated rule IDs"
+        id: find_ids
+        run: |
+          for file in ${{ steps.changed-files.outputs.all_changed_and_modified_files }}; do
+            echo "$file was changed"
+            rule_id=$(yq '.id' $file)
+          
+            echo "$file has rule ID $rule_id"
+            altered_rule_ids=$(echo "$rule_id"" ""$altered_rule_ids")
```

Comment:
This is so cursed `"$rule_id"" ""$altered_rule_ids"` but I don't know what else I expect from bash.

Reply:
Haha `"$rule_id $altered_rule_ids" should work too but I'll extra `"` wrap often in case I'm forgetting some expansion that's going to happen.